tools/lib/capitalization.py

```diff
@@ -161,14 +162,17 @@
 SPLIT_BOUNDARY = "?.!"  # Used to split string into sentences.
 SPLIT_BOUNDARY_REGEX = re.compile(rf"[{SPLIT_BOUNDARY}]")
 
+# Regexes which check capitalization in sentences.
```

Comment:
This comment is repeated twice, please recheck your code for simple errors as such to make it easier for the reviewers.

Reply:
Also, the only change that should have been part of this commit in the current file is the addition of **Jitsi** in the ignored phrases. If you want to do additional cleanup, please include it as an additional commit with a commit message that explains those changes.